File: test_api.js
// Test API calls from console
const testSignup = async () => {
    const data = {
        firstName: 'TuğbaTest',
        lastName: 'TestAdi',
        email: `test_${Date.now()}@example.com`,
        password: 'Test123456',
        phone: '5551234567'
    };
    
    console.log('Testing signup with:', data);
    try {
        const response = await fetch('http://localhost:5000/api/auth/signup', {
            method: 'POST',
            headers: { 'Content-Type': 'application/json' },
            body: JSON.stringify(data)
        });
        
        const result = await response.json();
        console.log('Response:', response.status, result);
        return result;
    } catch (error) {
        console.error('Error:', error);
    }
};

testSignup();

File: test_api.py
import requests

# Test if API is responding
try:
    r = requests.get('http://hesappaylas.com/api/test', timeout=10)
    print(f'Status: {r.status_code}')
    print(f'Content type: {r.headers.get("content-type")}')
    print(f'First 300 chars: {r.text[:300]}')
except Exception as e:
    print(f'Error: {e}')
